src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -224,7 +225,7 @@ foam.CLASS({
           categoryId));
     },
     function signingOfficerQuestion() {
-      return this.auth.check(this.__subContext__, "businessHasSigningOfficer");
+      return this.auth.check(this.ctrl.__subContext__, 'businessHasSigningOfficer');
```

Comment:
Assuming this is temporary?

Reply:
YES for sure, this will be a preRequisite of BusinessRegistration